### DISCUSSION TITLE
Documentation and default parameters do not match for micom.workflows.grow.grow

### DISCUSSION THREAD

**Discussion by anubhavdas0907** [micom-dev/micom#80] [General]
Hello Christian,

I just wanted to point out that in the documentation for micom.workflows.grow.grow, the default strategy in the code is 'minimal imports', while in the explanation, pFBA is mentioned as default. Kindly edit accordingly.

```
micom.workflows.grow.grow(manifest, model_folder, medium, tradeoff, threads=1, weights=None, strategy='minimal imports', atol=None, rtol=None, presolve=False)
```
strategy : str
        Computational strategy used to reduce the flux space. Default "pFBA" uses
        parsimonious FBA, ...

Regards
Anubhav

**Comment by cdiener** [comment]
Will do thanks!
